requirements.txt

```diff
@@ -1,9 +1,9 @@
-git+https://github.com/CitrineInformatics/taurus.git@develop
+taurus-citrine==0.1.0
```

Comment:
This will likely resolve https://github.com/CitrineInformatics/citrine-python/issues/65